src/arviz_plots/plots/bfplot.py

```diff
@@ -0,0 +1,229 @@
+        pc_kwargs["plot_grid_kws"] = pc_kwargs.get("plot_grid_kws", {}).copy()
+
+        pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+        pc_kwargs["aes"].setdefault("color", ["component_group"])
+
+
+        figsize = pc_kwargs["plot_grid_kws"].get("figsize", None)
+        figsize_units = pc_kwargs["plot_grid_kws"].get("figsize_units", "inches")
+        if figsize is None:
+            figsize = plot_bknd.scale_fig_size(
+                figsize,
+                rows=1,
+                cols=1,
+                figsize_units=figsize_units,
+            )
+            figsize_units = "dots"
+        pc_kwargs["plot_grid_kws"]["figsize"] = figsize
+        pc_kwargs["plot_grid_kws"]["figsize_units"] = figsize_units
+
+        plot_collection = PlotCollection.grid(
+            distribution,
+            backend=backend,
+            **pc_kwargs,
+        )
+
+    plot_kwargs.setdefault("credible_interval", False)
+    plot_kwargs.setdefault("point_estimate", False)
+    plot_kwargs.setdefault("point_estimate_text", False)
+
+    plot_collection = plot_dist(
+        distribution,
+        var_names=var_name,
+        group=None,
+        coords=None,
+        sample_dims=sample_dims,
+        kind=kind,
+        point_estimate=None,
+        ci_kind=None,
+        ci_prob=None,
+        plot_collection=plot_collection,
+        backend=backend,
+        labeller=labeller,
+        plot_kwargs=plot_kwargs,
+        stats_kwargs=stats_kwargs,
+        pc_kwargs=pc_kwargs,
+    )
+
+    if ref_val == 0:
+        ref_dt = xr.Dataset({var_name: xr.DataArray([ref_val])})
+        _, ref_aes, ref_ignore = filter_aes(plot_collection, aes_map, "ref_line", sample_dims)
+        if "color" not in ref_aes:
+            ref_line_kwargs.setdefault("color", "black")
+        if "linestyle" not in ref_aes:
+            default_linestyle = plot_bknd.get_default_aes("linestyle", 2, {})[1]
+            ref_line_kwargs.setdefault("linestyle", default_linestyle)
+        if "alpha" not in ref_aes:
+            ref_line_kwargs.setdefault("alpha", 0.5)
+
+        plot_collection.map(
+            vline, "ref_line", data=ref_dt, ignore_aes=ref_ignore, **ref_line_kwargs
+        )
+
+
+
+    bf_text_kwargs = copy(plot_kwargs.get("bf_text", {}))
+    if bf_text_kwargs is not False:
+        _, bf_text_aes, bf_text_ignore = filter_aes(
+            plot_collection, aes_map, "bf_text", sample_dims
+        )
+
+
+    bf_dt = xr.Dataset({var_name: xr.DataArray([0])})
+    plot_collection.map(
+        annotate_xy,
+        "bf_text",
+        text=f"BF10: {bf["BF10"]:.2f}\nBF01: {bf["BF01"]:.2f}",
+        x=ref_val,
+        y=max(ref_vals["prior"], ref_vals["posterior"]),
+        data=bf_dt,
+        ignore_aes=bf_text_ignore,
+        **bf_text_kwargs,
+    )
+
+    if backend == "matplotlib":  ## remove this when we have a better way to handle legends
+        plot_collection.add_legend("component_group", loc="outside right upper")
```

Comment:
I don't like the name "component_group" It is here just because I copied it from another plot where that name makes sense. Maybe just `group`, or just don't have a title for the legend...